oneanddone/users/views.py

```diff
@@ -151,6 +151,15 @@ def login_failure(self, *args, **kwargs):
             logging in, let us know by emailing <a href="mailto:{email}">{email}</a>.
         """).format(email='oneanddone@mozilla.com'), extra_tags='safe')
         return super(Verify, self).login_failure(*args, **kwargs)
+    
+    def login_success(self, *args, **kwargs):
+        if (not UserProfile.objects.filter(user=self.request.user).exists()):
+            self.success_url = reverse_lazy('users.profile.create')
+            return super(Verify, self).login_success(*args, **kwargs)
+        elif (not UserProfile.objects.get(user=self.request.user).privacy_policy_accepted):
```

Comment:
As above, this needs to use `self.user` and also you do not need the `(` and `)` around the condition here, so you should remove them.